pkg/security/ptracer/syscalls_arm64.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux && arm64
+
+package ptracer
+
+import (
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	OpenatNr   = 56  // OpenatNr defines the syscall ID for arm64
```

Comment:
shouldn't openat2 be part of this list ?

Reply:
nice catch, thanks